h/static/scripts/annotator/guest.coffee

```diff
@@ -417,6 +433,7 @@ module.exports = class Guest extends Annotator
     event.preventDefault?()
     event.stopPropagation?()
     @adder.hide()
+    console.log($(event.target).data('action'))
```

Comment:
This was left in by mistake. You might want to have a look at modifying our JSHint config file to catch such mistakes.